tests/resources/app_structure/suite.xml

```diff
@@ -61,6 +61,43 @@
       <datum id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='test_case'][@status='open']" value="./@case_id" detail-select="m0_case_short" detail-confirm="m0_case_long"/>
     </session>
   </entry>
+
+  <sync-request>
```

Comment:
Basic test that the parser works. More tests will come when the objects parsed start actually getting used.